packages/rpc/src/protocol.ts

```diff
@@ -737,5 +737,9 @@ export function rpcDecodeError(error: EncodedError): Error {
     const e = new Error(error.message);
     e.stack = error.stack + '\nat ___SERVER___';
 
+    if(error.stack === "") {
+        e.stack = ""
+    }
+
```

Comment:
this does not include errors that are nominally thrown like these (since it early exits at line 730)

```typescript
@entity.name('@error:user')
class UserNotFound extends Error{}
```

while looking at that code I really wonder if we get then even a good stack trace at all even within the client. it seems the new `e.stack = ''` and the old `e.stack = error.stack ...` just drops all the information, which is not good. we likely want to maintain the actual stack trace on the client side so users see in what code the error was thrown